src/scenes/Root/Sidebar/Sidebar.stories.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Sidebar } from './Sidebar';
+
+export default { title: 'Components/Sidebar' };
+
+export const SidebarStory = () => <Sidebar />;
```

Comment:
This shows up as `Components/Sidebar/Sidebar Story`. The extra nesting is redundant since there's only one and we don't want _Story_ in the name. Also it's under Scenes now.

```suggestion
export default { title: 'Scenes/Root' };

export const Sidebar = () => <Sidebar />;
```

Reply:
Moved to correct location